specificatie/kiesrecht.yaml

```diff
@@ -21,18 +21,20 @@ components:
           $ref: 'datum.yaml#/components/schemas/GbaDatum'
         einddatumUitsluitingKiesrecht:
           $ref: 'datum.yaml#/components/schemas/GbaDatum'
-    Kiesrecht:
+    UitsluitingKiesrecht:
       type: object
       properties:
-        europeesKiesrecht:
-          type: boolean
-          description: |
-            Geeft aan of persoon een oproep moet ontvangen voor verkiezingen voor het Europees parlement.
-          example: true
         uitgeslotenVanKiesrecht:
           type: boolean
           example: true
-        einddatumUitsluitingEuropeesKiesrecht:
-          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
         einddatumUitsluitingKiesrecht:
           $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    EuropeesKiesrecht:
+      type: object
+      description: |
+        * **europeesKiesrecht** - DGeeft aan of persoon een oproep moet ontvangen voor verkiezingen voor het Europees parlement. Wordt gevuld op basis van de waarden die voorkomen in de tabel 'Europees_Kiesrecht' uit de Haal-Centraal-BRP-tabellen-bevragen API.
+      properties:
+        europeesKiesrecht:
```

Comment:
we hebben hier herhaling van de groepsnaam het veld. Dus het pad is europeesKiesrecht.europeesKiesrecht.

Is het niet duidelijker dit te wijzigen naar "aanduiding" (volledig pad wordt dan europeesKiesrecht.aanduiding)? Het gaat immers om het element "Aanduiding Europees kiesrecht".

En het volgende veld wordt dan "einddatumUitsluiting" (volledig pad wordt dan europeesKiesrecht.einddatumUitsluiting)

Reply:
Aangepast